components/brave_wallet_ui/page/screens/onboarding/components/welcome-action/welcome-action.style.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import * as leo from '@brave/leo/tokens/css'
+import Icon from '@brave/leo/react/icon'
+
+import { WalletButton } from '../../../../../components/shared/style'
+
+export const WelcomeActionWrapper = styled(WalletButton)`
```

Comment:
Leo buttons have the challenge of styling the content you wrap with the button due to the shadow DOM. In case like this I prefer just using an HTML button or `WalletButton`. Mainly I just want the users to easily focus on the element with keyboard navigation and for accessibility.